scripts/wiki/ddl.sql

```diff
@@ -0,0 +1,55 @@
+-- DDL for parsed Wiki data.
+
+CREATE TABLE entities (
+    id TEXT PRIMARY KEY NOT NULL,
+    -- This could be normalized. Not worth it at the moment though, since they aren't used.
+    claims TEXT
+);
+
+-- The FTS5 virtual table implementation doesn't allow for indices, so we rely on ROWID to match entities.
+-- This isn't great, but with a controlled data ingestion setup this allows for stable matching.
+-- Same for foreign keys.
+CREATE VIRTUAL TABLE entities_texts USING fts5(
+    entity_id UNINDEXED,
+    name,
+    description,
+    label
+);
+
+CREATE TABLE articles (
+    entity_id TEXT PRIMARY KEY NOT NULL,
+    id TEXT NOT NULL,
+    FOREIGN KEY(entity_id) REFERENCES entities(id)
+);
+CREATE UNIQUE INDEX idx_articles_id
+ON articles (id);
+
+-- Same here: no indices possible, relying on ROWID to match with articles.
+CREATE VIRTUAL TABLE articles_texts USING fts5(
+    entity_id UNINDEXED,
+    title,
+    content
+);
+
+CREATE TABLE properties_in_entities (
+    property_id TEXT NOT NULL,
+    from_entity_id TEXT NOT NULL,
+    to_entity_id TEXT NOT NULL,
+    PRIMARY KEY (property_id, from_entity_id, to_entity_id),
```

Comment:
Is `property_id` not a foreign key on `entities` as well?

Reply:
No, `property_id` references Wikidata properties of which an entity can have arbitrarily many. We don't store properties in our DB since we don't use them as of now (this might change if we want to be able to consider ontologies though).